docs/src/content/docs/es/guides/components.mdx

```diff
@@ -173,6 +173,61 @@ import { LinkCard } from '@astrojs/starlight/components';
 	<LinkCard title="Componentes" href="/es/guides/components/" />
 </CardGrid>
 
+### Apartados
+
+Los apartados (también conocidos como "admoniciones" o "llamadas") son útiles para mostrar información secundaria junto al contenido principal de una página.
```

Comment:
I agree. I think it would be best to remove the parentheses 
```suggestion
Los apartados son útiles para mostrar información secundaria junto al contenido principal de una página.
```